test/nextflow.config

```diff
@@ -8,7 +8,7 @@ params {
     // Directories
     base_dir = "${launchDir}" // Parent for working and output directories (can be S3)
     raw_dir = "${launchDir}/raw" // Raw data directory
-    ref_dir = "s3://nao-mgs-wb/index-20240618/output/results" // Reference/index directory (generated by index workflow)
+    ref_dir = "s3://nao-mgs-wb/index-20240618/output" // Reference/index directory (generated by index workflow)
```

Comment:
This is the only non-comment/readme change in the PR. I had to make this change to get the test data to run with the existing index.